dmorg.tex

```diff
@@ -150,6 +150,32 @@ \subsubsection{Hardware Configuration Control}
 
 The sizing model for compute hardware purchasing is detailed in \citeds{LDM-144}, \citeds{LDM-141}, and \citeds{LDM-138}.
 
+\subsection {Release Management } \label{sect:release}
+
+
+The DMCCB maintains  the release plan, \citeds{LDM-564},  synchronized with  the project milestones.
+
+All releases will be identified by a release issue - but we wish to move to Jira release functionality.
+
+Any unscheduled  release, major, minor or patch, needs to be requested by the end user to the DMCCB using a RFC Jira issue. The RFC shall contain:
+
+\begin{itemize}
+\item The justification for  the release.
+\item The date the release is requested to be available.
+\item A list of proposed functionalities or fixes (Jira issues) which are requested to be implemented in the release.
+\end{itemize}
+
+The DMCCB will assess the release request within one week. If the release is urgent, DMCCB will assess it within 24 hours.
```

Comment:
Given that the DMCCB meets once a week, how will it address something within 24 hours? Does this just mean “the DMPM will unilaterally decide”, which is the DMCCB's fast decision making process? If so, let's write that.

(Even that seems ambitious... if somebody requests a release while Wil is on holiday, are we really going to demand he make a ruling within the day?)

Reply:
I suggest to add following sentence: 
"... within 24 hours. Procedure to be clarified in LDM-564."

Reply:
Actually, the right place where this shall be clarified, is not in the release document, but in the CCB section.